app/components/endpoints/endpointsController.js

```diff
@@ -45,6 +45,7 @@ function ($scope, $state, EndpointService, EndpointProvider, Notifications, Pagi
   $scope.addEndpoint = function() {
     var name = $scope.formValues.Name;
     var URL = $scope.formValues.URL;
```

Comment:
Let's use `var URL = $filter('stripprotocol')($scope.formValues.URL)` directly here and skip the declaration of `filteredURL`.